week06_policy_based/atari_wrappers.py

```diff
@@ -1,6 +1,9 @@
 """ Environment wrappers. """
 from collections import defaultdict, deque
 
+import multiprocessing
+multiprocessing.set_start_method("fork")
```

Comment:
Can you confirm you are trying to make this work on Mac OS? I'm confused because you mention Windows in your PR description while [Python docs](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.get_all_start_methods) say:

> On Windows only `'spawn'` is available.